documentation/RLC-Swagger-quickstart.md

```diff
@@ -138,24 +138,25 @@ We are working on to automatically generate everything right now, but currently
 
 # Improve README.md document
 
-A minimal README.md is generated by our codegen and you could improve it with more meaningful information.
+A minimal README.md is generated by our codegen and you could improve README.md file per your service. To learn more about README, see an [example README.md here](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/template/template/README-EXAMPLE.md).
 
 
 # How to write test for RLC
 
-In order to release it, we need to add some tests for it to make sure we are delivering high quality packages. But before we add the test, we need to add a `generate-test: true` make the codegen generate the necessary change in `package.json` and `tsconfig.json` so that test framework can work. Once the generation finished, you will see a `sampleTest.spec.ts` file in your `{PROJECT_ROOT}/test/public` folder, which only has an empty test and you may change them into test against your own services.
+In order to release it, we need to add some tests for it to make sure we are delivering high quality packages. But before we add the test, we need to enable the option `generate-test: true` to generate the necessary change in `package.json` and `tsconfig.json` so that test framework can work. Once the generation finished, you will see a `sampleTest.spec.ts` file in your `{PROJECT_ROOT}/test/public` folder, which has an empty test and you could add/update test cases against your own services.
```

Comment:
```suggestion
In order to release your RLC library, we need to add some tests for it to make sure we are delivering high quality packages. But before adding the test, we need to enable the option `generate-test: true` to generate the necessary change in `package.json` and `tsconfig.json` so that test framework can work. Once the generation finished, you will see a `sampleTest.spec.ts` file in your `{PROJECT_ROOT}/test/public` folder, which has an empty test and you could add/update test cases against your own services.
```